application/models/EmailModel.php

```diff
@@ -41,6 +41,7 @@ public function sendEmail($emailAddress, $paymentMethod)
             $TICK_IMAGE = "assets/images/Green_Tick.png";
             $PAYMENT_DETAIL = "PAID ONLINE";
             $MSG_COLOUR = "#00ff00";
+			$TRANSFER_DETAIL = "";
```

Comment:
Is the indentation correct?

Reply:
I'm not sure why GitHub have weird indentations. It's not showing the same indentation as when I did the commit.